app/services/import/persist/persist-company.service.js

```diff
@@ -0,0 +1,140 @@
+'use strict'
+
+/**
+ * Creates or updates an imported company and its child entities that have been transformed and validated
+ * @module PersistCompanyService
+ */
+
+const AddressModel = require('../../../models/address.model.js')
+const CompanyModel = require('../../../models/company.model.js')
+const ContactModel = require('../../../models/contact.model.js')
+const { db } = require('../../../../db/db.js')
+
+/**
+ * Creates or updates an imported company and its child entities that have been transformed and validated
+ *
+ * @param trx
+ * @param updatedAt
+ * @param {object} transformedCompanies - An object representing a valid WRLS licence
+ *
+ * @returns {Promise<string>} - the licence id from WRLS
```

Comment:
This isn't correct. But also aware it's returning something we don't expect any calling module to use.

```suggestion
 * @returns {Promise} A promise is returned but it does not resolve to anything we expect the caller to use
```